server/image_remove.go

```diff
@@ -69,7 +69,7 @@ func (s *Server) removeImage(ctx context.Context, imageRef string) (untagErr err
 
 		status, statusErr = s.ContainerServer.StorageImageServer().ImageStatusByName(s.config.SystemContext, name)
 		if statusErr != nil {
-			log.Errorf(ctx, "Error getting image status %s: %v", name, statusErr)
+			log.Debugf(ctx, "Error getting image status %s: %v", name, statusErr)
```

Comment:
Why did you change this debug?

Reply:
We need to fall through this case in case of an artifact, means we would indicate an error in the logs which can still succeed if the object is an artifact.

Reply:
I'm concerned that `statusErr` is something else than NotFound and not logged.
How about using `Warnf` or having a branch based on the error?

Reply:
Sounds good, let's use `Warnf`.